source/DSCResources/DSC_SqlSetup/DSC_SqlSetup.psm1

```diff
@@ -518,6 +529,11 @@ function Get-TargetResource
     .PARAMETER ProductKey
         Product key for licensed installations.
 
+   .PARAMETER PRODUCTCOVEREDBYSA
```

Comment:
Please set the correct casing of the parameter name. Also need an extra space before `.PARAMETER`